java/src/dev/selenium/tools/modules/ModuleGenerator.java

```diff
@@ -17,551 +17,180 @@
 
 package dev.selenium.tools.modules;
 
-import static com.github.javaparser.ParseStart.COMPILATION_UNIT;
-import static net.bytebuddy.jar.asm.Opcodes.ACC_MANDATED;
-import static net.bytebuddy.jar.asm.Opcodes.ACC_MODULE;
-import static net.bytebuddy.jar.asm.Opcodes.ACC_OPEN;
-import static net.bytebuddy.jar.asm.Opcodes.ACC_STATIC_PHASE;
-import static net.bytebuddy.jar.asm.Opcodes.ACC_TRANSITIVE;
-import static net.bytebuddy.jar.asm.Opcodes.ASM9;
-import static net.bytebuddy.jar.asm.Opcodes.V11;
-
-import com.github.bazelbuild.rules_jvm_external.zip.StableZipEntry;
-import com.github.javaparser.JavaParser;
-import com.github.javaparser.ParseResult;
-import com.github.javaparser.ParserConfiguration;
-import com.github.javaparser.Provider;
-import com.github.javaparser.Providers;
+import com.github.javaparser.*;
 import com.github.javaparser.ast.CompilationUnit;
-import com.github.javaparser.ast.Modifier;
-import com.github.javaparser.ast.NodeList;
 import com.github.javaparser.ast.expr.Name;
-import com.github.javaparser.ast.modules.ModuleDeclaration;
-import com.github.javaparser.ast.modules.ModuleExportsDirective;
-import com.github.javaparser.ast.modules.ModuleOpensDirective;
-import com.github.javaparser.ast.modules.ModuleProvidesDirective;
-import com.github.javaparser.ast.modules.ModuleRequiresDirective;
-import com.github.javaparser.ast.modules.ModuleUsesDirective;
-import com.github.javaparser.ast.visitor.VoidVisitorAdapter;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.PrintStream;
-import java.io.UncheckedIOException;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLClassLoader;
-import java.nio.file.FileVisitResult;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.nio.file.SimpleFileVisitor;
-import java.nio.file.StandardCopyOption;
+import com.github.javaparser.ast.modules.*;
```

Comment:
Please ensure formatting changes are done as part of a separate PR. Also, I suggest breaking down the PR into smaller changes that are grouped with a single motivation so that it is easy to review. Since this class impacts how CDP classes are generated and I don't think there is an easy way to test if there are any regressions with the changes made in this so we want to ensure no breaking changes go in.